README.md

```diff
@@ -1,141 +1,146 @@
-## 2.2 Liabilities
-Since agreements must be enforceable without violating the security guarantees of the underlying protocol, liability for any agreement is limited to the collateral that agents deposit when creating the agreement. We assume that any digital assets can be used as collateral, and define a general interface for defining the liability of an agreement.
- - **MIN\_LIABILITY**: defines collateral which is exclusively locked for the duration of an Agreement. It's availability as recourse for a dispute is guaranteed.  
- - **MAX\_LIABILITY**: defines collateral which is locked for the duration of an Agreement, but which can also be used to resolve disputes for other Agreements that share the same enforcement mechanism. Agreements which rely on MAX\_LIABILITY are making a collateral efficiency/enforceability tradeoff, that enables the emergence of a robust credit system.  
- - **FINALIZATION**: defines EVM code that will be executed to determine whether the agreement is still valid or has finalized. Allows composability of different criteria for an agreement to finalize. Time or block spans, the ability for both or a third party to end it or prolong it are basic examples of what could be computed here.  
+One block after the heartbeat function is executed for the term in which a dispute is scheduled, a function to draft jurors for that dispute is called. This function can be called at any point during the term and the resulting selection of jurors will be the same. If this transaction does not happen before the term ends the dispute must be rescheduled for a subsequent court term.
 
-## 2.3 Enforcement
-Agreements must also specify an address which all parties of the agreement grant privileges to re-assign deposited collateral in the event of a dispute. This can be the address of a trusted third party, but in practice we expect that most Agreements will be enforced by the Aragon Court mechanism due to the network effects and collateral efficiency of using MAX_LIABILITY Agreements.
-# 3. Courts
-Courts are the native enforcement mechanism for the Aragon Network, they provide users with a neutral shared context for Arbitration. Each court produces an oracle feed of judgments on subjective disputes. Over time, as disputes are successfully resolved and jurors earn reputation, the judgments of a court will become more consistent as precedents are set and reinforced.
+To make the draft function efficient, active jurors are arranged into a tree structure based on their stake. Using the random seed for the term and the id of the dispute a random number is generated and used to traverse the tree and arrive at a juror. This process is repeated until all juror slots for the dispute have been filled.
```

Comment:
```suggestion
To make the draft function efficient, active jurors are arranged into a tree structure based on their stake. Using the random seed for the term and the ID of the dispute, a random number is generated and used to traverse the tree and arrive at a juror. This process is repeated until all juror slots for the dispute have been filled.
```

Reply:
Is the "ID" a transaction ID?

Reply:
its unique identifier of the dispute (not transaction).